test/bddtests/restclient/sender.go

```diff
@@ -30,7 +31,12 @@ func SendRequest(url string, req []byte) (*HttpRespone, error) {
 
 // SendResolveRequest send a regular GET request to the sidetree-mock and expects 'side tree document' argument as a response
 func SendResolveRequest(url string) (*HttpRespone, error) {
-	client := &http.Client{}
+	client := &http.Client{
+		// TODO add tls config
+		// TODO !!!!!!!remove InsecureSkipVerify after configure tls for http client
```

Comment:
May be we can add a common trustbloc issue # to this TODO. It would be easy to search all with that issue.

Reply:
how

Reply:
we need to add it per project

Reply:
I add TODO

Reply:
Either an issue in one of the projects and resuse (not ideal), or creating an issue in each project when pushing the PR just not not lose track of it.